File: file_py/rata_rata.py
def rata(a):
    i = 1
    hasil = 0
    while (i <= a):
        b = int(input(f"Masukkan angka ke-{i} : "))
        hasil += b
        
        i += 1
        
    rata2 = hasil / a
    return rata2

a = int(input("Masukkan jumlah angka: "))
print(f"Hasil rata-rata dari {a} angka adalah {rata(a)}")

File: file_c_c++/rata_rata.c
#include <stdio.h>

float rata(int a) {
    int angka;
    int jumlah = 0;
    float rata2;
    for(int i = 1; i <= a; i++) {
        printf("Masukkan angka ke-%d : ", i, "\n");
        scanf("%d", &angka);
        jumlah = jumlah + angka;
    }
    
    rata2 = jumlah / a;
    return rata2;
}

int main() {
    int n;
    printf("Ingin masukkan berapa angka : ");
    scanf("%d", &n);
    printf("Rata-rata dari %d angka tersebut adalah : %f\n", n, rata(n));
    return 0;
}